includes/modules/sideboxes/featured_categories.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * featured sidebox - displays a random Featured Catelog
+ *
+ * @copyright Copyright 2003-2022 Zen Cart Development Team
+ * @copyright Portions Copyright 2003 osCommerce
+ * @license http://www.zen-cart.com/license/2_0.txt GNU Public License V2.0
+ * @version $Id: DrByte 2020 Jul 10 Modified in v1.5.8-alpha $
+ * based on featured and brands
+ */
+
```

Comment:
Overall, adhere to PSR12 indentation standards.  Code initially starts with no indentation, then 4 spaces for conditional clauses.